src/compute/src/bin/computed.rs

```diff
@@ -99,48 +100,21 @@ async fn main() {
     }
 }
 
-fn create_communication_config(args: &Args) -> Result<timely::CommunicationConfig, anyhow::Error> {
-    if args.addresses.len() > 1 {
-        let process = match args.process {
-            None => {
-                bail!(
-                    "--process argument must be specified when more than one address is specified"
-                );
-            }
-            Some(process) if process >= args.addresses.len() => {
-                bail!(
-                    "process index {process} out of range [0, {})",
-                    args.addresses.len()
-                );
-            }
-            Some(process) => process,
-        };
-        Ok(timely::CommunicationConfig::Cluster {
-            threads: args.workers,
-            process,
-            addresses: args.addresses.clone(),
-            report: true,
-            log_fn: Box::new(|_| None),
-        })
-    } else {
-        match args.process {
```

Comment:
Just reading, but this previously would allow us to hand in a `process` of `1` I think, which .. was probably a bug and it is fine that we are removing that possibility, right?

Reply:
yeah the logic as it was doesn't make sense to me. Seems like it was a mistake.